doc/Articles/Guides/Sync/SyncVars.md

```diff
@@ -1,69 +1,52 @@
-
-SyncVars can use any [type supported by Mirage](../DataTypes.md). You can have up to 64 SyncVars on a single NetworkBehaviour script, including SyncLists (see next section, below).
+SyncVars can use any [type supported by Mirage](../DataTypes.md). You can have up to 64 SyncVars on a single NetworkBehaviour script, including [SyncLists](SyncLists.md) and other sync types.
 
 The server automatically sends SyncVar updates when the value of a SyncVar changes, so you do not need to track when they change or send information about the changes yourself. Changing a value in the inspector will not trigger an update.
 
->   The [SyncVar hook](SyncVarHook.md) attribute can be used to specify a method to be called when the SyncVar changes value on the client.
-
-## SyncVar Example
-Let's say we have a networked object with a script called Enemy:
+## Example
+Let's have a simple `Player` class with the following code:
 
 ``` cs
-public class Enemy : NetworkBehaviour
-{
-    [SyncVar]
-    public int health = 100;
-
-    void OnMouseUp()
-    {
-        NetworkIdentity ni = NetworkClient.connection.identity;
-        PlayerController pc = ni.GetComponent<PlayerController>();
-        pc.currentTarget = gameObject;
-    }
-}
-```
+using Mirage;
+using UnityEngine;
 
-The `PlayerController` might look like this:
-
-``` cs
-public class PlayerController : NetworkBehaviour
+public class Player : NetworkBehaviour
 {
-    public GameObject currentTarget;
+    [SyncVar]
+    public int clickCount;
 
     void Update()
     {
-        if (isLocalPlayer)
-            if (currentTarget != null)
-                if (currentTarget.tag == "Enemy")
-                    if (Input.GetKeyDown(KeyCode.X))
-                        CmdShoot(currentTarget);
+        if (IsLocalPlayer && Input.GetMouseButtonDown(0))
+        {
+            IncreaseClicks();
+        }
     }
 
-    [Command]
-    public void CmdShoot(GameObject enemy)
+    [ServerRpc]
+    public void IncreaseClicks()
     {
-        // Do your own shot validation here because this runs on the server
-        enemy.GetComponent<Enemy>().health -= 5;
+        // This is executed on the server
+        clickCount++;
     }
 }
 ```
 
-In this example, when a Player clicks on an Enemy, the networked enemy game object is assigned to `PlayerController.currentTarget`. When the player presses X, with a correct target selected, that target is passed through a Command, which runs on the server, to decrement the `health` SyncVar. All clients will be updated with that new value. You can then have a UI on the enemy to show the current value.
+In this example, when Player A clicks the left mouse button, he sends a RPC to the server where the `clickCount` SyncVar is incremented. All other visible players will be informed about Player A's new `clickCount` value.
```

Comment:
please add a link to Server Rpc article on **RPC**

Reply:
Done.